app/assets/stylesheets/restaurants.scss

```diff
@@ -1,3 +1,112 @@
-// Place all the styles related to the Restaurants controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin make-float-boxes(){
+  float: left;
+  display: inline-block;
+}
+
+@mixin image-no-repeat($url){
+  background-image: url($url);
+  background-repeat: no-repeat;
+
+}
+.header{
+  position: relative;
+  width: 100%;
+  height: 200px;
+  background-size: cover;
+  @include image-no-repeat('header.png');
```

Comment:
I think this include is an overkill, especially if you give more background properties like background-size: cover, in this case `background: url(...) no-repeat left top; background-size: cover;` is good enough;